### PR TITLE
Enable logging-not-lazy lint check in travis CI

### PR DESCRIPTION
**Pull request by PaulYuuu**
Enable W1201 lint check in travis CI.

ID: 1925560
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Hello @luckyh , as we discussed offline, enable "W1201" check is helpful to standardize the call of "logging".
However, since `.foramt()` and fstring formatting can pin one to many, E.g: `"{0}-{1}-{0}".format(1, 2)`, so I did not enable 1202 and 1203 together. What's your opinion about this?

File: .travis.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@ install:
     - pip install -r requirements-travis.txt
 
 script:
-    - inspekt checkall --disable-style E501,E265,W601,E402,E722,E741 --no-license-check
+    - inspekt checkall --disable-style E501,E265,W601,E402,E722,E741 --enable-lint W0611,W1201 --no-license-check
```
Comment (luckyh) [inline .travis.yml:18]: I remember that `W0611` is enabled by default, is there some reason to do so?

**Comment by luckyh** [comment]
> Hello @luckyh , as we discussed offline, enable "W1201" check is helpful to standardize the call of "logging".
> However, since `.foramt()` and fstring formatting can pin one to many, E.g: `"{0}-{1}-{0}".format(1, 2)`, so I did not enable 1202 and 1203 together. What's your opinion about this?

Makes sense to me, thanks.

**Comment by PaulYuuu** [comment]
> > Hello @luckyh , as we discussed offline, enable "W1201" check is helpful to standardize the call of "logging".
> > However, since `.foramt()` and fstring formatting can pin one to many, E.g: `"{0}-{1}-{0}".format(1, 2)`, so I did not enable 1202 and 1203 together. What's your opinion about this?
> 
> Makes sense to me, thanks.

@luckyh okay, I will rebase to the latest code and check again, merge after CI is completed. Thank you for the review.

**Pull request merged**